widgets/cpu.js

```diff
@@ -9,7 +9,11 @@ calculateCPUPercent = function(statItem, previousCpu, previousSystem) {
     if (systemDelta > 0.0 && cpuDelta > 0.0) {
         cpuPercent = (cpuDelta / systemDelta) * statItem.cpu_stats.cpu_usage.percpu_usage.length * 100.0
     }
-    return cpuPercent
```

Comment:
I was seeing values of 200 to 20,000 here when selecting a second container. Needs a bit more debugging. Sometimes `(cpuDelta / systemDelta)` on line 10 is negative.